docs/available_commands.md

```diff
@@ -16,17 +16,130 @@ For more information on a particular operation, use the `cdx-ev amend --help-ope
 
 Note that the order of operations cannot be controlled. If you want to ensure two operations run in a certain order you must run the command twice, each time with a different set of operations.
 
-### Copy license texts from files
+### Insert license texts from files
 
-The program can copy the text describing a license from a specific file into the SBOM, if a license name is given.
+The operation `add-license-text` can be used to insert known full license texts for licenses identified by name. You can use this, for instance, in workflows where SBOMs are created or edited by hand - so a clutter-free JSON is preferred - then, in a last step, full texts are inserted using this operation.
 
-This is done by submitting the path to a folder containing txt-files with the license text via the command `--license-path`.
-If for example the license name "Apache License 1.0" is given, the program will search in the provided folder for the file "Apache License 1.0.txt" and copy its content in the `text` field.
-The txt-files in the folder must follow the naming convention name.txt.
+License texts are inserted only if:
 
-    cdx-ev amend bom.json" --license-path=C:\Documents\licenses
+* The license has a `name` field.
+* The license has no `id` field.
+* The license has no or an empty `text.content` field.
+* A matching file is found.
 
-If a `text` field already exists, its content will be replaced.
+You must provide one file per license text in a flat directory. The filename must match the license name specified in the SBOM. The filename's extension (anything after and including the last period) can be present in the license name but it doesn't need to.
```

Comment:
I would also mention how to submit the directory (--license-dir command) its not mentioned here or in help, i found it  through the error message (when providing it  wrong), so with just this documentaion i was unable to use the command.

Maybe add an example call of the function.

Reply:
@CBeck-96 can this conversation be resolved as per your [comment](https://github.com/Festo-se/cyclonedx-editor-validator/pull/185#issuecomment-2187385430)?